src/coreclr/jit/emitarm64sve.cpp

```diff
@@ -1926,22 +1926,17 @@ void emitter::emitInsSve_R_R(instruction     ins,
                 assert(insOptsScalableStandard(opt));
                 return emitInsSve_R_R_I(INS_sve_pmov, attr, reg1, reg2, 0, opt, sopt);
             }
-            if (sopt == INS_SCALABLE_OPTS_TO_PREDICATE)
```

Comment:
Are there any functions we can remove `sopt` completely?

Reply:
I think Alan means are there any `emitInsSve_*` methods that no longer need the `sopt` parameter, because it's only checking that `sopt` is `INS_SCALABLE_OPTS_NONE`? `emitInsSve_R_R` seems close to that, though we still check `sopt` with `insScalableOptsWithVectorLength` at the end of the method.

Reply:
Ah ok. Thanks @amanasifkhalid for clarifying. I didn't check, but can do it as a follow-up work.